src/helpers/momentHelper.js

```diff
@@ -74,3 +74,56 @@ export const isActive = (item) => {
 
   return true;
 };
+
+/**
+ *
+ * Find the item with the closest date range to today.
+ *
+ * @param {DataItem[]} data an array of objects, each with a `dates` array
+ *
+ * @return {DataItem | null} the item with the closest date range to today, or null if no such item exists
+ */
+export const findClosestItem = (data) => {
+  // Check if data is null or undefined or not an array
+  if (!Array.isArray(data)) {
+    return null;
+  }
```

Comment:
the comment does not fit the code, or am i wrong? this checks for "not an array" only?
i suggest not using a comment that translates the code in english. better would be a comment explaining what is it useful for?

Reply:
Yes, you're right, I'll update the comment line.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/909/commits/d664c3f0a762f98b5502f2da02380b3c1cbd2e5b